src/DynamoCore/Graph/Workspaces/SerializationExtensions.cs

```diff
@@ -29,7 +29,8 @@ internal static string ToJson(this WorkspaceModel workspace, EngineController en
                 Formatting = Formatting.Indented,
                 Converters = new List<JsonConverter>{
                         new ConnectorConverter(),                        
-                        new WorkspaceWriteConverter(engine)
+                        new WorkspaceWriteConverter(engine),
```

Comment:
different question here :  any reason for not using WorkspaceWriteConverter? WriteJSON method can validate this and take a different path. I could be totally wrong here.

Reply:
I went with a DummyNodeConverter because the WorkspaceConverter serializes the nodes as a collection, so it seemed to me to be a clean way to properly serialize a DummyNode to it's original content.